### ISSUE TITLE
🔨 Add session / stream / channel

### ISSUE THREAD

**Issue by jmsaulnier** [elastos/Elastos.NET.Carrier.Nodejs.SDK#35]
Latest commit added session / stream / channel

It would be great to add an example 

Thanks!

**Comment by jmsaulnier** [comment]
@kuit Pushed a demo here https://github.com/elastos/Elastos.NET.Carrier.Nodejs.SDK/commit/560392d01b32d9e97c22b2824347ab1e4f4f5b55

**Comment by kevingzhang** [comment]
@jmsaulnier  are you OK with the demo? If you do, please close this issue.

**Comment by jmsaulnier** [comment]
@kevingzhang do you think it would be possible to have an example like https://github.com/elastos/Elastos.Carrier.Demo.Remoter.iOS as well?
Or add a "video call" option in the chat demo, it should be possible right?

**Comment by kevingzhang** [comment]
Yes, we should. We should put this task into Bounty Program so that there would be someone work on it. The problem is that the bounty program is still under construction. We may have to do it offline. Can you make such a demo yourself? @jmsaulnier

**Comment by jmsaulnier** [comment]
@kevingzhang yes I can try!
